package.json

```diff
@@ -34,6 +34,7 @@
 		"social-previews:dev:build": "env-cmd -f ./src/environments/browser-path.env tsx watch --tsconfig tsconfig.script.json build-scripts/social-previews/live-server.ts",
 		"social-previews:dev:server": "env-cmd -f ./src/environments/browser-path.env live-server build-scripts/social-previews/dist",
 		"social-previews:dev": "npm-run-all --parallel social-previews:dev:build social-previews:dev:server",
+		"search-index:deploy": "tsx --env-file-if-exists=.env --tsconfig tsconfig.script.json build-scripts/search-index/deploy.ts",
```

Comment:
Now that we're using Node 22, we should eventually use experimental TS stripping from Node itself instead of `tsx`.